ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/smartcharging/GetCompositeScheduleConfirmation.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * GetCompositeScheduleConfirmation.java
+ *
+ * Created on 2019-11-21
+ *
+ * Copyright (C) 2019 Volkswagen AG, All rights reserved.
```

Comment:
By adding this in the library, it will automatically fall under the MIT license. Is it possible for you to add the MIT license explicitly in the source file, just to avoid misunderstandings?